romancal/regtest/test_wfi_image_pipeline.py

```diff
@@ -300,5 +300,5 @@ def test_pipeline_suffix(rtdata, ignore_asdf_paths):
         assert model.meta.cal_step.flat_field == "COMPLETE"
         assert model.meta.cal_step.photom == "COMPLETE"
         assert model.meta.cal_step.source_detection == "COMPLETE"
-        assert model.meta.cal_step.tweakreg == "INCOMPLETE"
+        assert model.meta.cal_step.tweakreg == "SKIPPED"
```

Comment:
This `assert` isn't currently reached in the regtests due to the `compare_against_truth` failing:
```
FAILED romancal/regtest/test_wfi_image_pipeline.py::test_pipeline_suffix - AssertionError: 
  Diff report for:
      result file: /runner/_work/_temp/pytest_basetemp/popen-gw2/test_pipeline_suffix0/r0000101001001001001_0001_wfi01_star.asdf
          model type: ImageModel
      truth file: /runner/_work/_temp/pytest_basetemp/popen-gw2/test_pipeline_suffix0/truth/r0000101001001001001_0001_wfi01_star.asdf
          model type: ImageModel
  
  {'dictionary_item_added': ["root['roman']['meta']['asn']", "root['roman']['meta']['exptype']", "root['roman']['meta']['group_id']"],
   'values_changed': {"root['roman']['meta']['cal_step']['tweakreg']": {'new_value': 'SKIPPED',
                                                                        'old_value': 'INCOMPLETE'}}}
```
But when 'okified' this change to the assert will be needed.